R-package/tests/testthat/test_basic.R

```diff
@@ -571,3 +571,26 @@ test_that("lgb.train() works with early stopping for regression", {
     , early_stopping_rounds + 1L
   )
 })
+
+test_that("lgb.train() supports non-ASCII feature names", {
```

Comment:
right here (inside `test_that()`) can you add `testthat::skip("UTF-8 feature names are not fully supported in the R package")`? That will leave the test but will skip it until I fix it.

Per, https://github.com/microsoft/LightGBM/pull/2976#issuecomment-610048511, let's skip R for now.  This isn't a regression...we didn't support it in R before and we still won't after this PR. I'll create a feature request for R

Reply:
Okay, done.

Reply:
thanks!

Reply:
@StrikerRUS @henry0312 I added #2983 to capture the work for R. We'll pick it up eventually, but since it would be net-new functionality I don't think it's as important as everything we're working on towards #629